core/src/lib.rs

```diff
@@ -213,6 +233,35 @@ impl Attribute {
         }
         flags
     }
+
+    pub fn to_edn_value(&self, ident: Option<&NamespacedKeyword>) -> edn::Value {
```

Comment:
Make that `Option<NamespacedKeyword>`, so you don't need to use `ident.clone()` on line 240; see below for the neat trick when calling it.